LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I think we can't provide a global license for the whole repository.
Each driver has it's own license.txt, and libraries also have their own license files (which are often not MIT).

If the community guidelines require a license file (and I would prefer to name it LICENSE.md), we can write an information that drivers and libraries each have their own license which can be found in the corresponding folder of the driver.

Reply:
That's a good point to add a LICENSE.md that makes once more clear that each driver comes with an individual license.